packages/esbuild-plugin/README.md

```diff
@@ -50,24 +50,105 @@ require("esbuild").build({
+| urlPrefix          | `string`                                                                   | optional | URL prefix to add to the beginning of all filenames. Defaults to `'~/'` but you might want to set this to the full URL. This is also useful if your files are stored in a sub folder. eg: url-prefix `'~/static/js'`.                                                                                                                                                                                                                                                                                                                                         |
+| urlSuffix          | `string`                                                                   | optional | URL suffix to add to the end of all filenames. Useful for appending query parameters.                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
+| validate           | `boolean`                                                                  | optional | When `true`, attempts source map validation before upload if rewriting is not enabled. It will spot a variety of issues with source maps and cancel the upload if any are found. Defaults to `false` as this can cause false positives.                                                                                                                                                                                                                                                                                                                       |
+| stripPrefix        | `array`                                                                    | optional | When paired with the `rewrite`, this will remove a prefix from filename references inside of sourcemaps. For instance you can use this to remove a path that is build machine specific. Note that this will NOT change the names of uploaded files.                                                                                                                                                                                                                                                                                                           |
```

Comment:
```suggestion
| stripPrefix        | `array`                                                                    | optional | When paired with the `rewrite` option, this will remove a prefix from filename references inside of sourcemaps. For instance you can use this to remove a path that is build machine specific. Note that this will NOT change the names of uploaded files.                                                                                                                                                                                                                                                                                                           |
```